vite.config.js

```diff
@@ -15,9 +23,9 @@ export default defineConfig({
   },
   build: {
     lib: {
-      entry: path.resolve(__dirname, 'src/index.js'),
+      entry: path.resolve(__dirname, 'src/index.ts'),
       name: 'Swimm UI',
-      fileName: (format) => `swimm-ui.${format}.js`,
+      fileName: (format) => `swimm-ui.${format}.ts`,
     },
     rollupOptions: {
       external: ['vue'],
```

Comment:
Make sure this is complete and there are no external deps bundled into the bundle that shouldn't be.

Reply:
As far as I know
@talkor ?

Reply:
Yeah I think @segevfiner has a point, as long as it's a dependency I think it's fine